src/hotspot/share/gc/parallel/parallelScavengeHeap.cpp

```diff
@@ -763,15 +753,96 @@ PSCardTable* ParallelScavengeHeap::card_table() {
   return static_cast<PSCardTable*>(barrier_set()->card_table());
 }
 
-void ParallelScavengeHeap::resize_young_gen(size_t eden_size,
-                                            size_t survivor_size) {
-  // Delegate the resize to the generation.
-  _young_gen->resize(eden_size, survivor_size);
+static size_t calculate_free_from_free_ratio_flag(size_t live, uintx free_percent) {
```

Comment:
Why refer to the `free_ratio_flag` instead of just `calculate_free_from_free_percent`?